editing/include/tests.js

```diff
@@ -5582,7 +5582,7 @@ function runConformanceTest(browserTest) {
 
             assert_equals(document.execCommand(browserTest[1][i][0], false, browserTest[1][i][1]),
                 expectedExecCommandReturnValues[i]);
-        }, testName + ": execCommand(" + format_value(browserTest[1][i][0]) + ", false, " + format_value(browserTest[1][i][1]) + ") return value");
+        }, testName + ": command #" + (i+1) + " execCommand(" + format_value(browserTest[1][i][0]) + ", false, " + format_value(browserTest[1][i][1]) + ") return value");
```

Comment:
While I'm confident that this change results in stable test names (I even wrote
up a whole response to that effect), the testing infrastructure is sufficiently
complex to make me less-than-certain. Since I'm highly averse to introducing
instability, I've explored more explicit approaches.

I've pushed up a commit that implements a more declarative mechanism. With this
latest version, test objects may optionally specify an arbitrary string with
any command. This allows the two offending tests in `multitest.js` to extend
the repeated "underline" commands with a comment (in this case, "first
application" and "second application").

This could certainly be done in an automated way, but (as stated earlier) I'm
reluctant to contribute to the complexity of the logic. For the time being,
the "repeated command" condition occurs so infrequently (i.e. just twice) that
the extra requirement seems acceptable.